feature-libs/order/styles/components/order-details/_order-details-items.scss

```diff
@@ -35,12 +35,6 @@
     padding: 0 30px;
   }
 
-  .cx-review-header {
```

Comment:
I think we need this. please double check

Reply:
Hi Wei, it seems styling for the `cx-review-header` has been already made in the _review.scss file or is this deliberate? The UI looks the same when added and removed.